packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/upgrade-guides.5/version-1-upgrade-guide.1.tool.md

```diff
@@ -317,7 +325,13 @@ npm run start-prod
 npm run start-static
 ```
 
-Note: If you don't have some of these commands it is not necessary to add them. Just ensure the commands you do have are working as expected. All of the `tt-serve` commands should be updated to use webpack-dev-server or the terra express server
+Note: If you don't have some of these commands it is not necessary to add them. If you changed from `tt-serve-static` to use the `express-server` command, be sure to also install the `@cerner/terra-open-source-scripts` package that provides this command. Just ensure the commands you do have are working as expected. All of the `tt-serve` commands should be updated to use webpack-dev-server or the terra express server.
```

Comment:
I wonder if we should split the express server out of open source scripts.. We didn't originally intend for internal teams to use that package.

Reply:
Move it to terra-functional-testing?

Reply:
I think we should move it to terra-functional-testing. We also forgot to add the @cerner/terra-open-source-scripts dependency when we did the upgrade in our other projects so we would need to go back to add the dependency if we don't split it.

Reply:
I'm onboard with moving it to terra-functional testing

Reply:
Ok. I'll move it as part of this PR and remove this from the doc.